docs/api-reference/web-mercator-viewport.md

```diff
@@ -50,9 +50,9 @@ using viewport projection parameters
 | `opts.topLeft` | `Boolean` | `true`   | If true projected coords are top left |
 
```

Comment:
(somehow cannot add comment on lines above)
Line 29-30 and 31-32 are duplicated, remove line 29-30.

Also, in line 32:
> - `[latitude, longitude]` can only be specified when `mercator` is true

where do we specify `mercator`?